File: POOUber/PHP/Card.php
<?php 
    
    require_once('Payment.php');
    class Card extends Payment {
        public $number;
        public $cvv;
        public $date;

        public function __constructor($id, $number, $cvv, $date){
            parent::__constructor($id);
            $this->number = $number;
            $this->cvv = $cvv;
            $this->date = $date;
        }
    }



?>

File: POOUber/Java/Card.java
package Java;

public class Card extends Payment{
    Integer numerCard;
    String dateCard;
    String cvv;

    public Card(Integer id, Integer numerCard, String dateCard, String cvv){
        super (id);
        this.numerCard = numerCard;
        this.dateCard = dateCard;
        this.cvv = cvv;
    }
}
